### PR TITLE
Continuity Updates

### PR DESCRIPTION
**Pull request by Nambaseking01**
Removed all outdated sentences and replaced wrong photos in forum helpers list.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,10 +17,7 @@ <h2>Who are the Scratch Forum Helpers?</h2>
...
 			<h2>I want to be a Forum Helper. How do I apply?</h2>
-			<p style="margin-left:200px;margin-right:200px">We're glad that you're interested in applying! We are always on the lookout for new members! However, there is a requirement, which is that you must have at least 100 posts to apply. In order to apply, please go to the Forum Helpers Studio (There is a link in the header), and fill out the application listed in the description of the studio.</p>
-			<h2>I submitted my application! Now what?</h2>
-			<p style="margin-left:200px;margin-right:200px">Once you submit an application, three of the Studio Managers will look at your application and grade it out of 10 points. The scores will be added together. You must get at least 24 to join. If you don't get the number, you may reapply in a week!</p>
-			<br>
+			<p style="margin-left:200px;margin-right:200px">You don't have to apply! The Forum Helpers handpick awesome forum helpers and invite them to the official studio!
```
Comment (Accio1) [inline index.html]: Please change this to say "There isn't an application process, but if three managers agree that you are being active and helpful on the Scratch Discussion Forums, they may choose to invite you to become a Forum Helper."

Comment (Nambaseking01) [reply index.html]: OK, I'll go ahead and change that up.

Comment (Nambaseking01) [reply index.html]: Finished!

**Comment by awesome5185** [comment]
Commenting for future reference.

**Comment by Nambaseking01** [comment]
@Accio1 Someone told me it would be a good idea to open the Forum Helpers studio when clicking on the button in a new tab so I'm going to update that. If you don't want it, I'll undo the commit.

**Comment by Accio1** [comment]
Merged!

**Pull request merged**